### ISSUE TITLE
Different build environments

### ISSUE THREAD

**Issue by boris-petrov** [kaliber5/ember-cli-bundlesize#11]
Hi, thanks for the great plugin!

We have a couple of production environments that we build (for example we build one for old ES5 browsers and one for newer ES6+ browsers). I'm not sure what exactly does `ember-cli-bundlesize` do when "creating a production build" - does it do something like `ember build --environment=production`? If so, is it possible to change that `environment` value? And how do we tell `ember-cli-bundlesize` the sizes for files in each of the builds?

Thanks!

**Comment by simonihmig** [comment]
> Hi, thanks for the great plugin!

Thanks! :)

> does it do something like ember build --environment=production

Yes, exactly: https://github.com/kaliber5/ember-cli-bundlesize/blob/master/lib/helpers/ember-build.js#L6

> If so, is it possible to change that environment value? And how do we tell ember-cli-bundlesize the sizes for files in each of the builds?

There is no specific support for this, but I think this should be possible by leveraging the CLI options `---build-app=false` and `--config-path=...`, see the Readme!

```bash
# test with --prod and default config
ember bundlesize:test
# test with different env and separate config
ember build --environment=es5
ember bundlesize:test --build-app=false --config-path=./config/bundlesize-es5.js
```

**Comment by boris-petrov** [comment]
@simonihmig - thanks for the answer. Two notes/questions:

1) When I do what you suggest in the last two lines of your example - does `ember-cli-bundlesize` use the built app in the `dist` folder? Is this also configurable or it always looks there?

2) The only negative part of doing it this way is that I have to have two configuration files for the two builds we have - not a big deal, just not very "neat".

Thanks again. I'll leave you to close the issue if you decide that there is nothing else to be done in this regard.

**Comment by boris-petrov** [comment]
Another thing that I saw as an issue is that `--config-path` expects a relative path and doesn't work with absolute ones.

**Comment by simonihmig** [comment]
> I can see here that the dist folder is hardcoded. Would you be willing to make this configurable?

Sure, that is fine. My time is currently very limited, but if you are willing to provide a PR, that would be great. Should be pretty easy, with a new `--build-dir` option in `availableOptions` in the same file.

> Another thing that I saw as an issue is that --config-path expects a relative path and doesn't work with absolute ones.

Hm, looking at https://github.com/kaliber5/ember-cli-bundlesize/blob/master/lib/tasks/test.js#L54, this seems to be true. Should be considered a bug. Can you open up another issue for this? A PR would be highly welcome as well of course! :wink:

**Comment by boris-petrov** [comment]
PR's created and awaiting review! :)

**Comment by simonihmig** [comment]
> PR's created and awaiting review!

Oh, wow, that was faaast! 😀Thank you!

**Comment by simonihmig** [comment]
Published as v0.2.1! 🎉

**Comment by boris-petrov** [comment]
Thank you, works like a charm!
